lib/pharos/cluster_manager.rb

```diff
@@ -36,7 +36,8 @@ def addon_manager
     # load phases/addons
     def load
       Pharos::PhaseManager.load_phases(__dir__ + '/phases/')
-      Pharos::AddonManager.load_addons(__dir__ + '/addons/')
+      addon_dirs = [__dir__ + '/addons/', File.join(Dir.pwd, 'extras')]
+      Pharos::AddonManager.load_addons(*addon_dirs)
```

Comment:
The `*addon_dirs` indirection seems unnecessary, just `load_addons(__dir__ + '/addons/', File.join(Dir.pwd, 'extras'))`?